bcgov/GDX-Analytics
# fix for alerts (#152-154 -157-160-161)

This PR updates the package-lock.json with the new versions of the following packages:
"ws" to "7.5.10"
"micromatch" to "4.0.8"
"webpack" to "5.94.0"
"fast-xml-parser" to "4.5.0"

This should solve the dependabot alerts [#152](https://github.com/bcgov/GDX-Analytics/security/dependabot/152) [#154 ](https://github.com/bcgov/GDX-Analytics/security/dependabot/154) [#157](https://github.com/bcgov/GDX-Analytics/security/dependabot/157) [#160](https://github.com/bcgov/GDX-Analytics/security/dependabot/160) [#161](https://github.com/bcgov/GDX-Analytics/security/dependabot/161) for the examples/vue_js_tracker.

Testing instructions:
- clone the repo and switch to branch
```
git clone https://github.com/bcgov/GDX-Analytics.git
cd GDX-Analytics/
git checkout GDXDSD-7143-vuejs-tracker-npm-install
```
- install vue and template compiler globally
```
npm install -g @vue/cli
npm i -g vue-template-compiler
```
- install the dependencies listed in the package-lock.json
```
cd examples/vue_js_tracker/
npm ci
```
- start a development server
```
npm run serve
```
check if sample vue.js page is running by browsing to http://localhost:8080/
